web/init/package.json

```diff
@@ -31,7 +32,8 @@
     "redux-segment": "^1.6.2",
     "redux-thunk": "2.2.0",
     "replicated-lint": "^0.12.12",
-    "yaml-ast-parser": "^0.0.41"
+    "yaml-ast-parser": "^0.0.41",
```

Comment:
What is this for?

Reply:
Same here, not sure, I didn't specifically install it